### DISCUSSION TITLE
MMixing antler and blade templates

### DISCUSSION THREAD

**Discussion by barrydevt** [statamic/cms#6212] [Q&A] [answered]
I trying out Statamic. I created a basic blog which is working great. Now I would like to add in a custom login page and some custom functionality 'outside' of statamic. 

So I created a custom route:

```
Route::get('/login', function(){
    return view('auth.login');
})->name('login');
```

I created a basic login form in the file `resources/views/auth/login.antlers.html`

But the view does not render any of the `layout`. 

The docs touch on this slightly at https://statamic.dev/blade#layouts where it implies that I should use a blade layout file here. 

This mixing of the template engines seems a bit messy, is there any way to just 'inherit' the antler layout in a detached antler template so I do not need to have a mishmash of blade and antler templates?

**Comment by duncanmcclean** [comment, marked as answer]
If you want Statamic's layouts system to work, you have to return like this instead (obviously replace the layout/template/data):

```php
return (new \Statamic\View\View)
        ->layout('layout')
        ->template('posts.category')
        ->with($category->toAugmentedArray());
```

**Reply by barrydevt** [reply]
Perfect, that's what I was looking for. Thank you.

**Reply by duncanmcclean** [reply]
No worries - have a great evening! 🙌
